app/controllers/application_controller.rb

```diff
@@ -1,6 +1,5 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+  include DeviseTokenAuth::Concerns::SetUserByToken
```

Comment:
Do we need this in the ApplicationController?  Will the token authorization be used in the 'normal' part of the application?